cmd/goal/node.go

```diff
@@ -595,6 +597,10 @@ var createCmd = &cobra.Command{
 		secondChoicePath := filepath.Join("var", "lib", "algorand", "genesis", newNodeNetwork, "genesis.json")
 		thirdChoicePath := filepath.Join(exePath, "genesisfiles", "genesis", newNodeNetwork, "genesis.json")
 		paths := []string{firstChoicePath, secondChoicePath, thirdChoicePath}
+		if gopath := os.Getenv("GOPATH"); len(gopath) > 0 {
+			fourthChoicePath := filepath.Join(gopath, "src", "github.com", "algorand", "go-algorand", "installer", "genesis", newNodeNetwork, "genesis.json")
+			paths = append(paths, fourthChoicePath)
+		}
```

Comment:
I'm not a big fan of this approach. The goal should come with pre-baked knowledge of the 3 supported networks, plus the ability to specify a custom genesis file. There shouldn't be any "search" done.

Reply:
I kind of agree, the GOPATH approach not ideal but at least it makes `goal node create` working on developers machines why do not use updater/packages.